### ISSUE TITLE
[MozFest] add copy for CC license + link to Press Center Page

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#4927]
Implementation ticket for footer changes.

**Impacted sites:**
[MozFest site](https://www.mozillafestival.org/en/)

**Changes:**
1. Removed "License" copy
2. Organized items by alphabetical order (except for "Donate", which should remain the first link)
2. Added "Most content available under a Creative Commons license." Link for [Creative Commons license is here](https://foundation.mozilla.org/about/website-licensing/)
3. Changed Mozilla logo to the most updated one. [Logo here](https://drive.google.com/drive/folders/1S8P67EV-c426se-eEdgS-cUorjqZI7EQ?usp=sharing)
4. Decreased distance between line and message "Mozilla is a global..." from 3rem to 1rem - "Nice to have" change. If it is too much trouble it is okay to leave it as is.
6. Replace "Media Center" copy. Use "Press Center" instead. Keep the [same link](https://www.mozillafestival.org/media/) as we currently have.

<img width="1171" alt="image" src="https://user-images.githubusercontent.com/22157921/87984494-e555e580-ca8e-11ea-88bf-7c310e531d1e.png">

Note: once this change is landed, it will unblock this ticket - #4825 

---

**🗒  Design -> Dev Handoff Checklist**
(Feel free to remove items that are not applicable.)

- [x] Design has been finalized
- [x] Link to original design ticket #4868
- [x] Link to InVision: [link here](https://mozilla.invisionapp.com/share/PJXWEC7BU86#/425165976_Footers-Mozfest)
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included
- [x] Assets are included

cc: @TheoChevalier - updates we will be doing to our English Footers
